docs/source/concepts/Ragged_Workspace.rst

```diff
@@ -0,0 +1,19 @@
+.. _Ragged_Workspace:
+
+Ragged Workspace
+=================
+
+What are they?
+--------------
+
+A ragged workspace is one where the spectra within it do not share a common set of x values. Ragged workspaces commonly occur after Converting the X units using ConvertUnits or after DiffractionFocussing.
+
```

Comment:
Nice to have this as a concept page!
For algorithms you can add a link to their doc page like this: 
```
:ref:`ConvertUnites <algm-ConvertUnits>`,
```
I'd suggest adding it for the algorithms mentioned here: ConvertUnits, DiffractionFocussing, Rebin (which in the old wiki page were broken links). When this is merged the old wiki page should be removed.